enhancements/samples/removed-bootstrapping.md

```diff
@@ -0,0 +1,188 @@
+- [/] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+
+## Summary
+
+In such scenarios where samples installation is doomed to fail out of the gate, it now is apparent to us that 
+the samples operator should start off a `Removed`, with no samples content installed.
+
+Those scenarios currently include
+- disconnect / restricted network installs
+- IPv6 single stack installs
+
+## Motivation
+
+With the mandate in 4.2 that the  samples operator should mark itself `Degraded` if any imagestream imports
+are still failing after 2 hours from the start of the install, cluster administrators have been caught off 
+guard when the samples operator eventually marks itself `Degraded` in such cases where samples installation 
+is doomed to fail without manual intervention from the cluster administrator.
+
+In particular, the options available are:
+- mirror desired imagestream images (where typically there are multiple versions per imagestream) to 
+an available and functional registry; and mirror them in such a way such that the repository and image spec
+match what is used on registry.redhat.io
+- since there are many imagestreams, most likely add the imagestreams they are not concerned with to the 
+skip list in the operator config object
+- override the registry used by the samples operator in its config object
+- or if you want to punt on samples entirely, mark the operator as `Removed`
+
+### Goals
+
+1) Alleviate out of the install concerns in scenarios where samples installation will fail out of the gate
+
+2) Allow the cluster administrator time to decide what to do with samples in the cluster
+
+### Non-Goals
+
+Other usability improvements around installing samples from registries other than registry.redhat.io have 
+been identified in Jira:
+
+- regular expressions for further manipulation of the image reference in the imagestream
+- modes where failed imagestreams are automatically added to the skip list (where the administrator then
+automatically can have them added to the management set by fixing the import by mirroring the images, fixing
+the registry in question, etc.)
+- moving the samples operator out of the payload / CVO and making it an OLM operator, where the cluster
+administrator controls which imagestreams are installed when the provision the OLM operator
+- aggregating network topology details like disconnected/mirrored via our cluster config API (which was previously 
+discussed and ultimately abandoned).
+
+Bootstrapping as removed is considered the meets minimum, least impacting change to the samples operator 
+that addresses the scenarios where out of the box imagestreams pointing to registry.redhat.io will fail.
+
+## Proposal
+
+With startup today, the samples operator bootstraps as `Managed`.  That boostrapping can be made conditional,
+and instead come up as `Removed` when the samples operator sees
```

Comment:
sees what?

Reply:
will change `sees` to `determines image stream imports will fail`